esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -29,7 +29,12 @@
             ESMVALTOOL_DIR = ${CYLC_WORKFLOW_RUN_DIR}/lib/ESMValTool
             PYTHONPATH_PREPEND = ${ESMVALCORE_DIR}:${ESMVALTOOL_DIR}
 
-    [[install_cold]]
+    [[install_env_file]]
+        platform = localhost
+        [[[environment]]]
+            SITE = {{ SITE }}
+
+    [[clone_latest_esmval]]
         platform = localhost
         [[[environment]]]
             SITE = {{ SITE }}
```

Comment:
These lines aren't needed for `clone_latest_esmval`, since this task now no longer uses the `SITE`. Would it be possible to remove these lines, please?

Reply:
9badc1f1de8439d30e570854752b328b3896ae00